src/access/tests/TestAccessControl.cpp

```diff
@@ -1037,22 +1032,24 @@ void TestSubjectsTargets(nlTestSuite * inSuite, void * inContext)
                    target.flags == (Target::kCluster | Target::kDeviceType) && target.cluster == 0xAAAA5555 &&
                        target.deviceType == 0xBBBB6666);
     NL_TEST_ASSERT(inSuite, entry.GetTarget(2, target) != CHIP_NO_ERROR);
+#endif
 }
 
 void TestUpdateEntry(nlTestSuite * inSuite, void * inContext)
 {
+#if 0
```

Comment:
`// TODO:(#issue)` (can be consolidated into one issue)

Reply:
No I'll be fixing these before merge.